src/pages/editor.js

```diff
@@ -134,27 +134,28 @@ const StoryblokEntry = (props) => {
    */
   useEffect(() => {
     // One time load only.
-    if (!mounted) {
-      // Storyblok Preview API access key.
-      const key = getParam('access_key');
-
-      // Must have the API Access key.
-      if (key.length === 0 || typeof key !== 'string') {
-        return;
+    const handleAccessToken = async () => {
+      try {
+        if (!mounted) {
+          // Storyblok Preview API access key.
+          const key = await getParam('access_key');
```

Comment:
If we add the accessToken here, do we still need L141? 🤔 I can do some additional clean up if we are able to utilize `process.env.GATSBY_STORYBLOK_ACCESS_TOKEN` instead.

Reply:
You can get the access_key from the URL parameters that are configured in storyblok's visual editor. If you need a preview key anywhere, that is where you'll need to get it from.

Reply:
So, the initBridge AND the StoryblokClient need to be initiated with the key. That isn't how their documentation is written but I don't want to fight it if it is working.

Reply:
Right? It bizzare that I had to piece this together from the clues they provided us in the support ticket and cross-reference one of their documentation. This ended up being a shot in the dark 🤨